### PR TITLE
feature: AutoGluon 0.8.2 image_uris update

### PR DESCRIPTION
**Pull request by yinweisu**
*Issue #, if available:*

*Description of changes:*
Add AutoGluon 0.8.2 image_uris metadata

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [x] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by tonyhoo — APPROVED** [review]

**Review by knikure — COMMENTED** [review]
/bot run all

File: tests/unit/sagemaker/image_uris/test_autogluon.py [hunk 1]
```diff
@@ -60,6 +60,8 @@
     "0.6.2",
     "0.7.0",
     "0.7",
+    "0.8",
```
Comment (akrishna1995) [inline tests/unit/sagemaker/image_uris/test_autogluon.py:63]: Can you please run `tox -e black-check` 

ref: https://github.com/aws/sagemaker-python-sdk/blob/master/.githooks/pre-push

**Review by akrishna1995 — COMMENTED** [review]
/bot run slow-tests,notebook-tests

**Review by knikure — COMMENTED** [review]
/bot run all

File: src/sagemaker/image_uri_config/autogluon.json [hunk 2]
```diff
@@ -345,6 +346,41 @@
                 "repository": "autogluon-training",
                 "processors": ["cpu", "gpu"],
                 "py_versions": ["py39"]
+            },
+            "0.8.2": {
```
Comment (akrishna1995) [inline src/sagemaker/image_uri_config/autogluon.json:350]: Please update issue description , Merge Checklist  etc 

Changes are LGTM at high level. PR tests and Notebook test failure unrelated to this change, tracking internally

File: tests/unit/sagemaker/image_uris/test_autogluon.py (hunk 1 above)
Comment (yinweisu) [reply tests/unit/sagemaker/image_uris/test_autogluon.py:63]: Addressed

File: src/sagemaker/image_uri_config/autogluon.json (hunk 2 above)
Comment (yinweisu) [reply src/sagemaker/image_uri_config/autogluon.json:350]: Updated

**Review by akrishna1995 — COMMENTED** [review]
/bot run all

**Review by akrishna1995 — COMMENTED** [review]
/bot run unit-tests

**Comment by akrishna1995** [comment]
PR and notebook tests are failing due to an existing issue. LGTM - merging the change

**Review by akrishna1995 — APPROVED** [review]
LGTM

**Pull request merged**